examples/processors/filter.tf

```diff
@@ -0,0 +1,52 @@
+terraform {
+  required_providers {
+    mezmo = {
+      source = "registry.terraform.io/mezmo/mezmo"
+    }
+  }
+  required_version = ">= 1.1.0"
+}
+
+provider "mezmo" {
+  auth_key = "my secret"
+}
+
+resource "mezmo_pipeline" "pipeline1" {
+  title = "My pipeline"
+}
+
+resource "mezmo_demo_source" "source1" {
+  pipeline_id = mezmo_pipeline.pipeline1.id
+  title       = "My source"
+  format      = "apache_common"
+}
+
+resource "mezmo_filter_processor" "processor1" {
+  pipeline_id = mezmo_pipeline.pipeline1.id
+  title       = "My processor"
+  description = "This processor filters logs"
+  inputs      = [mezmo_demo_source.source1.id]
+  conditionals = {
+    expressions = [
+      {
+        field        = ".status"
+        operator     = "greater_or_equal"
+        value_number = 300
+      },
+      {
+        field        = ".level"
+        operator     = "greater_or_equal"
+        value_string = "info"
+      }
+    ]
+    logical_operation = "OR"
+  }
+}
+
+resource "mezmo_logs_destination" "destination1" {
+  pipeline_id   = mezmo_pipeline.pipeline1.id
+  title         = "My destination"
+  description   = "Send logs to Mezmo Log Analysis"
+  inputs        = [mezmo_filter_processor.processor1.id]
+  ingestion_key = var.my_ingestion_key
```

Comment:
I don't think we need this.  It doesn't add value for understanding the new processor, and it's another place we'd have to update if the mezmo_logs destination ever changes.